src/coreclr/jit/codegenarm64.cpp

```diff
@@ -6702,6 +6702,14 @@ void CodeGen::genArm64EmitterUnitTests()
     theEmitter->emitIns_R_R_R(INS_asrv, EA_4BYTE, REG_R8, REG_R9, REG_R10);
     theEmitter->emitIns_R_R_R(INS_rorv, EA_4BYTE, REG_R8, REG_R9, REG_R10);
 
+    // TODO-SVE: Fix once we add predicate registers
+    theEmitter->emitIns_R_R_R(INS_uzp1, EA_8BYTE, REG_R0, REG_R1, REG_R2,
```

Comment:
This is emitting a neon `uzp1`, it should be `INS_sve_uzp1`

Reply:
good catch. i will address this and fix the tool.

Reply:
With the above change, you're going to have to add `case INS_sve_uzp1` into `emitIns_R_R_R()`

Reply:
yes and many such cases in `emitIns*` methods.